src/game.js

```diff
@@ -0,0 +1,64 @@
+class Game {
+  constructor(frameClass = Frame) {
+    this.frameClass = frameClass
+    this.frames = [];
+    this.outputArray = [];
```

Comment:
Does this need to be an instance variable created here? Or can it be a local variable that is just returned from the `cumulative()` function?

Having as little state as possible reduces the risk that we accidentally manipulate this variable in a method